frontend/main/components/NavigationMenu/NavigationMenu.js

```diff
@@ -78,6 +77,14 @@ export class NavigationMenu extends BaseComponent {
       const button = event.target.closest(".nav-button");
       if (button) {
```

Comment:
Can you make marketplace active when first loading and when clicking on "Aquatica"?